Dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM google/debian:wheezy
+FROM gcr.io/google_appengine/base
```

Comment:
@dlorenc is there a debian base under gcr.io? It seems weird to use the google_appengine base here.